localflavor/br/forms.py

```diff
@@ -178,3 +179,34 @@ def clean(self, value):
             raise ValidationError(self.error_messages['invalid'])
 
         return orig_value
+
+
+class BRBankChoiceField(Field):
```

Comment:
I think it's better use `ChoiceField` which would really simplify the code. I think doing something like this should work:

```
class BRBankChoiceField(ChoiceField):
    def __init__(self, **kwargs):
        super().__init__(**kwargs)
        self.choices = BR_BANK_CHOICES
```

You can override the `invalid_choice` key in `default_error_messages` if you want a different error message.